test/pytorch_test_base.py

```diff
@@ -460,7 +460,7 @@ def _alt_lookup(d, keys, defval):
   # Overrides to instantiate tests that are known to run quickly
   # and correctly on XLA.
   @classmethod
-  def instantiate_test(cls, name, test):
+  def instantiate_test(cls, name, test, *, generic_cls):
```

Comment:
With this change, we can probably remove the `if-else` block here

https://github.com/pytorch/pytorch/blob/46595a962397b10715076b658786c55e3dcfe109/torch/testing/_internal/common_device_type.py#L594-L600